sp800-63a/sec5_proofing.md

```diff
@@ -47,7 +47,7 @@ The following table lists qualities, ranging from unacceptable to superior, of i
 
 |Strength|Method(s)|
 |:---:|:------------------------------| 
-|Unacceptable|Validation of the evidence failed.|
+|Unacceptable|No validation or validation of the evidence failed.|
```

Comment:
Reads awkwardly. Perhaps either add comma after first *validation* or change to "Validation of the evidence was not performed or failed."

Analogous comment on next diff chunk.